tests/nvme_test.py

```diff
@@ -147,22 +147,23 @@ def nvme_reset_ctrl(self):
         self.assertEqual(proc.wait(), 0, "ERROR : pci rescan failed")
 
     def get_ctrl_id(self):
-        """ Wrapper for extracting the controller id.
+        """ Wrapper for extracting the first controller id.
             - Args:
                 - None
             - Returns:
                 - controller id.
         """
-        get_ctrl_id = "nvme list-ctrl " + self.ctrl
+        get_ctrl_id = f"nvme list-ctrl {self.ctrl} --output json"
```

Comment:
hmm, should be `--output-format` IIRC. Does `--output` work for you? I remember we did some changes in the option parsing section. Did we break something again?

Reply:
interesting `--output` works as well. Maybe the parser is a bit funky and matches it with `--output-format`. Anyway, use `--output-format` and we need look into why `--output` is also working. Yay another bug!

Reply:
I changed it to `--ouput-format=json` for all occasions :)
I can start looking at the faulty parsing next.